src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -53,6 +57,11 @@ public boolean ballIsPresent() {
     public void stop() {
       m_intakeMotor.stopMotor();
     }
-   
-
+    
+    public boolean ballIsNotPresent() {
```

Comment:
No need for this method. You can just use `(! ballIsPresent())` in the IsFinished method of VomitBall.
Alternatively keep this method but use 
`return  (! ballIsPresent())`

Reply:
I made the change